config/samples/v1alpha1_clustertemplate.yaml

```diff
@@ -1,17 +1,20 @@
 apiVersion: o2ims.provisioning.oran.org/v1alpha1
 kind: ClusterTemplate
 metadata:
-  labels:
-    app.kubernetes.io/name: clustertemplate
-    app.kubernetes.io/instance: clustertemplate-sample
-    app.kubernetes.io/part-of: oran-o2ims
-    app.kubernetes.io/managed-by: kustomize
-    app.kubernetes.io/created-by: oran-o2ims
   name: clustertemplate-sample.v1.0.0
+  namespace: clustertemplate-a-v4-16
 spec:
+  name: clustertemplate-a 
+  description:   nice description
+  templateId: 57b39bda-ac56-4143-9b10-d1a71517d04f
```

Comment:
Probably we should remove templateId since it's auto-generated

Reply:
Fixed